tests/test_parallel_corpus.py

```diff
@@ -0,0 +1,155 @@
+import os
+import subprocess
+from io import StringIO
+
+import pytest
+import sentencepiece
+
+from exquisite_corpus.parallel_corpus import (
+    cleanup_parallel_file, decode_pieces_with_sp, encode_with_sp_as_pieces,
+    get_vocabulary_from_sp, train_sentencepiece
+)
+
+DECODED = [
+    'abcdefghijklmnopqrstuvwxyz',
+    'this is a sample text',
+    ''
```

Comment:
Why is there a third item here that doesn't correspond to anything in ENCODED, and seems to not be used? Am I missing where it's used?

Reply:
Ah, I think it is a leftover- I will remove it.